foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -748,9 +851,65 @@ The Android SDK automatically registers an `Application.ActivityLifecycleCallbac
 * Tracking Deep Link click-through.
 * Tracking Push Message click-through
 * Tracking Local Notification click-through
+
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+**Objective-C**
+
+ This method should be called to support the following use cases:
+
+ * Tracking Deep Link click-throughs
```

Comment:
"Tracking deep link clickthroughs" - clickthroughs is one word, and we don't need to capitalize deep link.

Reply:
The comment about clickthrough applies to all mentions.